src/reducers/operations.ts

```diff
@@ -93,19 +102,19 @@ export const operationsReducer =
     };
 
 /**
- * A middleware that resets a failed action after timeout.
+ * A middleware that cancels a failed/successful action after timeout.
  * It respects AsyncAction.ttl if it's defined.
- * @param ttl default timeout before resetting a failed action. zero means: do not reset.
+ * @param ttl default timeout before cancelling a failed action. zero means: do not cancel.
  */
-export const failureResetter =
+export const actionCanceler =
   (ttl: number = 5000): Middleware =>
     (store) => (next) => (action: any) => {
 
-      if (isAsyncAction(action) && action.sequence == AsyncActionSequence.Failure) {
+      if (isAsyncAction(action) && (action.sequence === AsyncActionSequence.Failure || action.sequence === AsyncActionSequence.Success)) {
```

Comment:
this times out all errors? lets remove that so that people will definitely see errors

Reply:
fixed: https://github.com/daostack/alchemy/pull/51/commits/5e4ff2d5e8d6ed8193c8b7a40775adaa472e7b02